libs/@guardian/source-react-components/scripts/create-icons/get-svgs-from-figma.ts

```diff
@@ -31,15 +30,21 @@ const FIGMA_API_OPTIONS = {
 	},
 };
 
+const figmaApi = async <T>(url: string): Promise<T> => {
+	const response = await fetch(
+		`https://api.figma.com/v1/${url}`,
+		FIGMA_API_OPTIONS,
+	);
+	const data = await response.json();
+	return data;
```

Comment:
I think this would achieve the same thing, as promises are always unwrapped:

```suggestion
	return response.json();
```